### PR TITLE
Update to README for Ubuntu 172.x.x.x

### PR DESCRIPTION
**Pull request by joshongithub**
Update to README to include Note and example for Ubuntu with host IP of 172.x.x.x

# Reason for This PR

`[Author TODO: add issue # or explain reasoning.]`

## Description of Changes

`[Author TODO: add description of changes.]`

- [ ] This functionality can be added in [`rust-vmm`](https://github.com/rust-vmm).

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The reason for this PR is clearly provided (issue no. or explanation).
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes are reflected in `firecracker/swagger.yaml`.
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

### CODE REVIEW

File: docs/network-setup.md [hunk 1]
```diff
@@ -28,6 +28,37 @@ sudo iptables -A FORWARD -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
...
+tap0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
+        inet 192.16.0.1  netmask 255.255.255.0  broadcast 0.0.0.0
+        inet6 fe80::d042:74ff:fec0:d4dc  prefixlen 64  scopeid 0x20<link>
+        ether d2:42:74:c0:d4:dc  txqueuelen 1000  (Ethernet)
+        RX packets 98  bytes 7434 (7.4 KB)
+        RX errors 0  dropped 0  overruns 0  frame 0
+        TX packets 38  bytes 3006 (3.0 KB)
+        TX errors 0  dropped 4 overruns 0  carrier 0  collisions 0
+```
+
```
Comment (serban300) [inline docs/network-setup.md:61]: Personally I find this section too verbose. I would just drop a mention that the code provided above is just an example and that the IP of the tap device should be chosen such that it's not in the same subnet as the IP address of the host.

**Comment by dianpopa** [comment]
Hi @joshongithub ,

In order to get your PR merged you also need to sign your commit (make sure you have 'Signed-off-by' in your commit body) and limit your markdown lines to 80 chars max.

**Comment by serban300** [comment]
Hi @joshongithub ! Are you still interested in contributing here ? Do you need any help in addressing the comments ?

**Comment by joshongithub** [comment]
I added this based on a note from a team member. If it doesn't meet the content criteria or submission workflow, feel free to reject the PR. 

Josh.

**Comment by serban300** [comment]
> I added this based on a note from a team member. If it doesn't meet the content criteria or submission workflow, feel free to reject the PR.
> 
> Josh.

Ok. @firecracker-microvm/compute-capsule does anyone have any concern about closing this PR ?

**Comment by alindima** [comment]
> > I added this based on a note from a team member. If it doesn't meet the content criteria or submission workflow, feel free to reject the PR.
> > Josh.
> 
> Ok. @firecracker-microvm/compute-capsule does anyone have any concern about closing this PR ?

I don't, but I think adding a note as you mentioned, that `the IP of the TAP device should be chosen such that it's not in the same subnet as the IP address of the host.` would be helpful. We can have a separate issue/PR

**Comment by sandreim** [comment]
> > > I added this based on a note from a team member. If it doesn't meet the content criteria or submission workflow, feel free to reject the PR.
> > > Josh.
> > 
> > 
> > Ok. @firecracker-microvm/compute-capsule does anyone have any concern about closing this PR ?
> 
> I don't, but I think adding a note as you mentioned, that `the IP of the TAP device should be chosen such that it's not in the same subnet as the IP address of the host.` would be helpful. We can have a separate issue/PR

I've created #2620.

**Pull request closed without merging**